### PR TITLE
[hi] Localize self-healing.md

### LINKED ISSUE #1593: [hi] Localize self-healing.md to Hindi
content: https://github.com/cncf/glossary/blob/main/content/en/self-healing.md

### PR DESCRIPTION
**Pull request by ShivangShandilya**
Signed-off-by: Shivang Shandilya <101946115+ShivangShandilya@users.noreply.github.com>

### Describe your changes

This PR adds self-healing.md in Hindi Language

### Related issue number or link (ex: `resolves #issue-number`)

Resolves #1593

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/self-healing.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
```
Comment (aj11anuj) [inline content/hi/self-healing.md:3]:
```suggestion
status: completed

```

File: content/hi/self-healing.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
+category: गुण
```
Comment (aj11anuj) [inline content/hi/self-healing.md:4]:
```suggestion
category: property

```

File: content/hi/self-healing.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
```
Comment (aj11anuj) [inline content/hi/self-healing.md:2]:
```suggestion
title: स्व-उपचार (Self Healing)

```

File: content/hi/self-healing.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
+category: गुण
+---
+
+एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम है।
```
Comment (aj11anuj) [inline content/hi/self-healing.md:7]:
```suggestion
एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम होती है। 

```

File: content/hi/self-healing.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
+category: गुण
+---
+
+एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम है। 
+इसमें एक "अभिसरण" या "नियंत्रण" लूप है जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।
```
Comment (aj11anuj) [inline content/hi/self-healing.md:8]:
```suggestion
इसमें एक "अभिसरण" या "नियंत्रण" लूप होता है, जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।

```

File: content/hi/self-healing.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
+category: गुण
+---
+
+एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम है। 
+इसमें एक "अभिसरण" या "नियंत्रण" लूप है जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।
+यदि कोई अंतर है (उदाहरण के लिए, वांछित से कम एप्लिकेशन इंस्टेंस चल रहे हैं),
```
Comment (aj11anuj) [inline content/hi/self-healing.md:9]:
```suggestion
यदि कोई अंतर होता है (उदाहरण के लिए, वांछित से कम एप्लिकेशन के इंस्टेंस चल रहे हैं), तो 

```

File: content/hi/self-healing.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+title: सेलफ हीलिंग (Self Healing)
+status: पूरित
+category: गुण
+---
+
+एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम है। 
+इसमें एक "अभिसरण" या "नियंत्रण" लूप है जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।
+यदि कोई अंतर है (उदाहरण के लिए, वांछित से कम एप्लिकेशन इंस्टेंस चल रहे हैं),
+यह सुधारात्मक कार्रवाई करेगा (उदाहरण के लिए, नए उदाहरण प्रारंभ करें)।
```
Comment (aj11anuj) [inline content/hi/self-healing.md:10]:
```suggestion
यह सुधारात्मक कार्रवाई करेगा (उदाहरण के लिए, नए एप्लीकेशन इंस्टैंस प्रारंभ करन)।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

File: content/hi/self-healing.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+---
+
+एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम होती है। 
+
+इसमें एक "अभिसरण" या "नियंत्रण" लूप होता है, जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।
+
+यदि कोई अंतर होता है (उदाहरण के लिए, वांछित से कम एप्लिकेशन के इंस्टेंस चल रहे हैं), तो 
+
+यह सुधारात्मक कार्रवाई करेगा (उदाहरण के लिए, नए एप्लीकेशन इंस्टैंस प्रारंभ करन)।
```
Comment (aj11anuj) [inline content/hi/self-healing.md:16]:
```suggestion
---
title: स्व-उपचार (Self Healing)
status: completed
category: property
tags: ["infrastructure", "property"]
---

एक स्व-उपचार प्रणाली बिना किसी मानवीय हस्तक्षेप के कुछ प्रकार की विफलताओं से उबरने में सक्षम होती है। 
इसमें एक "अभिसरण" या "नियंत्रण" लूप होता है, जो सिस्टम की वास्तविक स्थिति को सक्रिय रूप से देखता है और इसकी तुलना उस स्थिति से करता है जिसे ऑपरेटर शुरू में चाहते थे।
यदि कोई अंतर होता है (उदाहरण के लिए, वांछित से कम एप्लिकेशन के इंस्टेंस चल रहे हैं), तो 
यह सुधारात्मक कार्रवाई करेगा (उदाहरण के लिए, नए एप्लीकेशन इंस्टैंस प्रारंभ करन)।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Comment by aj11anuj** [comment]
@ShivangShandilya you need to change target branch from 'cncf: main' to 'cncf: dev-hi'.

**Review by bishal7679 — APPROVED** [review]
LGTM!
Thank you @ShivangShandilya

**Pull request merged**